sim/midas/src/main/cc/bridges/fpga_managed_stream.h

```diff
@@ -6,7 +6,25 @@
 #include <functional>
 #include <string>
 
-#include "bridge_stream_driver.h"
+#include "managed_stream.h"
+
+class FPGAManagedStreamIO {
```

Comment:
Adding class docs explaining that the host-platform library needs to implement these would be helpful.